File: fibonacci/fibonacci.py
def get_fibonacci_series(num):
     series = [0, 1]
     while len(series) < num:
          series.append(series[len(series)-1] + series[len(series)-2])
     return series

print(f"Fibonacci series length - 8: {get_fibonacci_series(8)}")

File: fibonacci/fibonacci.go
package main

import "fmt"

func main() {
	fmt.Printf("Fibonacci series of 5: %v\n", getFibonacciSeries(5))
	fmt.Printf("Fibonacci series of 10: %v\n", getFibonacciSeries(10))
}

func getFibonacciSeries(num uint) []uint {
	series := []uint{0, 1}
	for i := 2; i < int(num); i++ {
		series = append(series, series[i-1]+series[i-2])
	}
	return series
}
